Components/Utils/ModelBuilder.cs

```diff
@@ -332,10 +335,23 @@ public static string BuildLanguageLinkURL(NameValueCollection nameValues)
                     ? Constants.FRENCH_CULTURE
                     : Constants.ENGLISH_CULTURE);
 
-            string url = string.Concat("?", Uri.EscapeUriString(nameValues.ToString()));
+            NameValueCollection queryStringValues = new NameValueCollection();
```

Comment:
Just noticed, forgot to cleanup one of our tests... `queryStringValues` does nothing... :)

Reply:
done!